src/pages/tasks/NewTaskPage.js

```diff
@@ -0,0 +1,211 @@
+        PropTypes.shape({
+            /** Display name of the person */
+            displayName: PropTypes.string,
+
+            /** Avatar URL of the person */
+            avatar: PropTypes.string,
+
+            /** Login of the person */
+            login: PropTypes.string,
+        }),
+    ),
+
+    /** Current user session */
+    session: PropTypes.shape({
+        email: PropTypes.string.isRequired,
+    }),
+
+    /** All reports shared with the user */
+    reports: PropTypes.objectOf(reportPropTypes),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    betas: [],
+    task: {},
+    personalDetails: {},
+    reports: {},
+    session: {},
+};
+
+/**
+ * Get the assignee data
+ *
+ * @param {Object} details
+ * @returns {Object}
+ */
+function constructAssignee(details) {
+    return {
+        icons: [{source: ReportUtils.getAvatar(lodashGet(details, 'avatar', ''), lodashGet(details, 'login', '')), type: 'avatar', name: details.login}],
+        displayName: details.displayName,
+        subtitle: details.login,
+    };
+}
+
+/**
+ * Get the share destination data
+ * @param {Object} reportID
+ * @param {Object} reports
+ * @param {Object} personalDetails
+ * @param {Object} policies
+ * @returns {Object}
+ * */
+function constructShareDestination(reportID, reports, personalDetails, policies) {
+    const report = lodashGet(reports, `report_${reportID}`, {});
+    return {
+        icons: ReportUtils.getIcons(report, personalDetails, policies),
+        displayName: ReportUtils.getReportName(report, policies),
+        subtitle: ReportUtils.getChatRoomSubtitle(report, policies),
+    };
+}
+
+// NOTE: This page is going to be updated in https://github.com/Expensify/App/issues/16855, this is just a placeholder for now
+const NewTaskPage = (props) => {
+    const [assignee, setAssignee] = React.useState({});
+    const [shareDestination, setShareDestination] = React.useState({});
+    const [submitError, setSubmitError] = React.useState(false);
+    const [parentReport, setParentReport] = React.useState({});
+
+    useEffect(() => {
+        setSubmitError(false);
+        if (props.task.assignee) {
+            const assigneeDetails = lodashGet(props.personalDetails, props.task.assignee);
+            const displayDetails = constructAssignee(assigneeDetails);
+            setAssignee(displayDetails);
+        }
+        if (props.task.parentReportID) {
+            setParentReport(lodashGet(props.reports, `report_${props.task.parentReportID}`, {}));
+            TaskUtils.setShareDestinationValue(props.task.parentReportID);
+        }
+        if (props.task.shareDestination) {
+            setParentReport(lodashGet(props.reports, `report_${props.task.shareDestination}`, {}));
+            const displayDetails = constructShareDestination(props.task.shareDestination, props.reports, props.personalDetails, props.policies);
+            setShareDestination(displayDetails);
+        }
```

Comment:
I'm a little confused, the shareDestination is the same as the parentReportID isn't it?